lib/OpenCloud/Database/Resource/Instance.php

```diff
@@ -190,13 +201,21 @@ protected function createJson()
             );
         }
 
-        return (object) array(
-            'instance' => (object) array(
+        $out = array(
```

Comment:
This is just a nit pick, but since we bumped the minimum PHP version to 5.4 recently, we can use short array syntax :)